src/style/utils/element-sizing.ts

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line no-restricted-imports
```

Comment:
**suggestion**: this line doesn't seem to raise this same linting error when this comment is removed.


```suggestion
```